### PR TITLE
Migrated phpcs to sniffs 3

### PR DESCRIPTION
**Pull request by mickel1138**
Fixed up .travis a bit.

### CODE REVIEW

**Comment by mickel1138** [comment]
https://github.com/squizlabs/PHP_CodeSniffer/wiki/Version-3.0-Upgrade-Guide#upgrading-custom-sniffs

**Comment by mickel1138** [comment]
PHPCS now working, try yourself:
./vendor/bin/phpcs --colors --standard=coding-standards/GRIIDC --extensions=php -s -n ./src

File: .env.test [hunk 1]
```diff
@@ -3,3 +3,4 @@ KERNEL_CLASS='App\Kernel'
 APP_SECRET='$ecretf0rt3st'
 SYMFONY_DEPRECATIONS_HELPER=999999
 PANTHER_APP_ENV=panther
+DATABASE_URL=postgres://postgres@127.0.0.1:3306/postgres
```
Comment (fightingtexasaggie) [inline .env.test]: please use localhost instead of 127.0.0.1. It's a separate db permission.

Comment (fightingtexasaggie) [inline .env.test]: 3306 is the default port on mysql. 5432 is the one for postgres

Comment (mickel1138) [reply .env.test]: Yes, that should be changed, but we are not creating a DB this time around, hopefully someday soon we can add fixtures and do proper regressions testing against the UI also.

Comment (fightingtexasaggie) [reply .env.test]: ok, good point...

File: .env.test [hunk 2]
```diff
@@ -3,3 +3,4 @@ KERNEL_CLASS='App\Kernel'
 APP_SECRET='$ecretf0rt3st'
 SYMFONY_DEPRECATIONS_HELPER=999999
 PANTHER_APP_ENV=panther
+DATABASE_URL=postgres://postgres@127.0.0.1:5432/postgres
```
Comment (praneethpr) [inline .env.test]: Can be removed

File: src/Controller/Api/DataCenterController.php [hunk 3]
```diff
@@ -157,7 +157,7 @@ public function getCollectionAction(Request $request)
     /**
      * Get a data center for a given id.
      *
-     * @param string $id The id of the Data center.
+     * @param integer $id The id of the Data center.
```
Comment (mickel1138) [inline src/Controller/Api/DataCenterController.php]: I changed this from string to integer, with the assumption that an ID always an integer is.

File: src/Entity/DIF.php [hunk 4]
```diff
@@ -635,7 +635,7 @@ public function getAbstract()
      *
      * @return void
      */
-    public function setFieldOfStudyEcologicalBiological($fieldOfStudyEcologicalBiological)
+    public function setFieldOfStudyEcologicalBiological(bool $fieldOfStudyEcologicalBiological)
     {
         if ('boolean' !== gettype($fieldOfStudyEcologicalBiological)) {
```
Comment (mickel1138) [inline src/Entity/DIF.php]: I guess this won't be needed anymore...

These entities are littered with checks, they should be done by Assert or Assert\Type if needed.

**Comment by mickel1138** [comment]
https://github.com/squizlabs/PHP_CodeSniffer/wiki/Advanced-Usage#ignoring-parts-of-a-file

New ignore syntax.

**Comment by mickel1138** [comment]
There are TWO coding standard issues remaining:
/src/Controller/GmlController.php has a non-nullable parameter at the end the function. (That's a no-no).
/src/Command/MakeDRPMCommand.php need cleanup. @fightingtexasaggie isle one!

File: src/Controller/Api/DataCenterController.php (hunk 3 above)
Comment (mickel1138) [reply src/Controller/Api/DataCenterController.php]: This will be a refactor.

File: src/Entity/DIF.php (hunk 4 above)
Comment (mickel1138) [reply src/Entity/DIF.php]: Will remove in a later PR.

**Pull request merged**